docs/source/tutorials/token_classification.ipynb

```diff
@@ -277,6 +288,7 @@
    "metadata": {},
```

Comment:
if we keep current structure, shouldn't this cell be hidden too since it's part of pulldown above?